test/router.js

```diff
@@ -983,8 +984,153 @@ describe('Router', function () {
       .expect(200, 'saw GET /bar', done)
     })
   })
+
+  describe('events', function () {
+
```

Comment:
These tests have a very inconsistent style from all other tests if you can clean them up at some point :)